organization/privacy.md

```diff
@@ -16,15 +16,16 @@ For people who have signed up to get our news, we collect:
 
 * name
 * email address
-* job title if applicable
-* organization if applicable
+* job title if applicable and provided
+* organization if applicable and provided
```

Comment:
Would we possibly track this even if this is not provided by them and we've looked this up ourselves?

Reply:
I haven't done this yet for the mailing list - would you?

Reply:
@MirjamvT and I note that our mailing list is not our CRM. Anyone we have legitimate business with should be in the CRM. Compiling a dossier on someone who's joined our mailing list would be creepy.

Reply:
@bvhme Please review this again!